templates/wizards/projects/asn1acn/wizard.json

```diff
@@ -0,0 +1,134 @@
+                                "value": "qmake",
+                                "condition": "%{JS: [ %{Plugins} ].indexOf('QmakeProjectManager') >= 0}"
+                            },
+                            {
+                                "trKey": "CMake",
+                                "value": "cmake",
+                                "condition": "%{JS: [ %{Plugins} ].indexOf('CMakeProjectManager') >= 0}"
+                            }
+                        ]
+                    }
+                },
+				{
+                    "name": "Sp1",
+                    "type": "Spacer",
+                    "data": { "factor": 4 }
+                },
+                {
+                    "name": "AddAsnFile",
+                    "trDisplayName": "Add ASN.1 file",
+                    "type": "CheckBox",
+                    "data": { "checked": false }
+                },
+                {
+                    "name": "AsnModelExtension",
+                    "trDisplayName": "Extension:",
+                    "type": "ComboBox",
+                    "visible": "%{JS: '%{AddAsnFile}'}",
+                    "data":
+                    {
+                        "index": 0,
+                        "items":
+                        [
+                            { "trKey": ".asn", "value": ".asn" },
+                            { "trKey": ".asn1", "value": ".asn1" }
+                        ]
+                    }
+                },
+                {
+                    "name": "AddAcnFile",
+                    "trDisplayName": "Add ACN file",
+                    "type": "CheckBox",
+                    "data": { "checked": false }
+                }
+            ]
+        },
+        {
+            "trDisplayName": "Project Management",
+            "trShortTitle": "Summary",
+            "typeId": "Summary"
+        }
+    ],
+    "generators":
+    [
+        {
+            "typeId": "File",
+            "data":
+            [
+                {
+                    "source": "file.pro",
+                    "target": "%{ProFile}",
+                    "openAsProject": true,
+                    "condition": "%{JS: '%{BuildSystem}' === 'qmake'}"
+                },
+                {
+                    "source": "CMakeLists.txt",
+                    "openAsProject": true,
+                    "condition": "%{JS: '%{BuildSystem}' === 'cmake'}"
+                },
+                {
+                    "source": "template.asn",
+                    "target": "%{AsnFullFileName}",
+                    "openInEditor": true,
+                    "condition": "%{AddAsnFile}"
+                },
+                {
+                    "source": "template.acn",
+                    "target": "%{AcnFullFileName}",
+                    "openInEditor": false,
+                    "condition": "%{AddAcnFile}"
+                },
+                {
+                    "source": "../git.ignore",
```

Comment:
Hmm, there common .gitignore file which is already provided in upper directory. This file is shared amon all the projects. I do not know yet if custom one would be needed.